osgtest/tests/test_155_stashcache.py

```diff
@@ -77,10 +79,15 @@
 """
 
 CACHE_AUTHFILE_PATH = PARAMS["StashCacheAuthfile"]
-CACHE_AUTHFILE_CONTENTS = "u * / rl\n"
+CACHE_AUTHFILE_CONTENTS = "u b64f6609.0 /osgtest/PROTECTED rl\n"
```

Comment:
What's "b64f6609.0"?

Reply:
It is the hash of the vdttest user cert.

Reply:
Is it guaranteed to be the same? If so, could you put a constant somewhere or add a comment or something?

Reply:
I added a comment and a constant.